content/ui-patterns/messaging.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Messaging
+---
+
+import {Box, Flex, BorderBox, Text} from '@primer/components'
+
+Messaging components are used to provide important and relevant information to the user, including feedback, contextual information, product updates, and more.
+
+Primer includes three different messaging components:
+
+- [Toasts](https://primer.style/css/components/toasts)
+- [Flash Alerts](https://primer.style/css/components/alerts)
+- [Popovers](https://primer.style/css/components/popover)
+
+## Messaging Types
+It's important to consider the context when selecting a component to convey your message. Does the message need to interrupt the user flow? Is this a global message that needs to be shown on every page? Does it require the user to take an action, or is it more of a notice? Most messaging on GitHub falls into one of the following categories:
+
+**Feedback**: communicating something to the user based on an action they've taken or a change in context
+- _Toasts or Flash Alerts are best suited for this type of messaging._
+
+**Highlights**: bringing attention to how and why to engage with a feature
+- _Popovers are best suited for this type of messaging._
+
+**Updates**: communicating necessary updates or changes to the product as it relates to the user
+- _Flash Alerts are best suited for this type of messaging._
+
+## Toasts
+Toasts are used to provide relevant feedback to the user, typically after they've taken an action. Primer includes styles for success, warning, error, loading, and default Toasts. Some examples of usage include:
+
+- confirmation that an action was successfully taken
+- communicating that an action is pending
+- notifying the user if an action failed to complete.
+- providing general information.
+
+Toasts are best used in context of the page they're referring to (rather than a global notification that can appear on any page) and should require minimal user interaction. For global notices and messaging, see the Flash component.
```

Comment:
```suggestion
Toasts are best used in context of the page they're referring to (rather than a global notification that can appear on any page) and should require minimal user interaction. For global notices and messaging, see the [Flash](#flash-alerts) component.
```
Should we cross-link?

Reply:
yeah totally, good call